test/atomics-unshared.wast.from-wast

```diff
@@ -3,7 +3,7 @@
  (memory $0 1 1)
  (func $foo
   (drop
-   (i32.atomic.rmw.cmpxchg
+   (i32.atomic.rmw.cmpxchg $0
```

Comment:
To reduce the test diff, can we avoid printing the name when there is a single memory?

Reply:
Done! 🎉